operations/spire/xname_validation.md

```diff
@@ -0,0 +1,207 @@
+         |  \
+         |___\      Shipping your Helm workloads with Loftsman
+       \--||___/
+  ~~~~~~\_____/~~~~~~~
+
+2022-03-29T18:15:22Z INF Ensuring that the loftsman namespace exists command=ship
+2022-03-29T18:15:22Z INF Loftsman will use the packaged charts at /etc/cray/upgrade/csm/csm-1.2.5/tarball/csm-1.2.5/helm as the Helm install source command=ship
+2022-03-29T18:15:22Z INF Running a release for the provided manifest at /tmp/tmp.rsDPG4zZf6/manifest.yaml command=ship
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Releasing spire v2.3.1
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+[TRUNCATED]
+
+2022-03-29T18:15:28Z INF Recording log data to configmap loftsman-xnamevalidation-ship-log in namespace loftsman command=ship
+deployment.apps/cray-opa-ingressgateway restarted
+deployment.apps/cray-opa-ingressgateway-customer-admin restarted
+deployment.apps/cray-opa-ingressgateway-customer-user restarted
+Warning: resource secrets/site-init is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
+secret/site-init configured
+spire-server is not ready. Will retry after 30 seconds. (1/30)
+[TRUNCATED]
+spire-server is not ready. Will retry after 30 seconds. (7/30)
+Enabling spire on NCNs
+Warning: Permanently added 'ncn-m001,10.252.1.13' (ECDSA) to the list of known hosts.
+[TRUNCATED]
+ncn-s001 is being joined to spire.
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100    86  100    54  100    32    637    377 --:--:-- --:--:-- --:--:--  1023
+Warning: Permanently added 'ncn-s001,10.252.1.6' (ECDSA) to the list of known hosts.
+[TRUNCATED]
+ncn-s002 is being joined to spire.
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100    86  100    54  100    32    623    369 --:--:-- --:--:-- --:--:--  1000
+Warning: Permanently added 'ncn-s002,10.252.1.5' (ECDSA) to the list of known hosts.
+[TRUNCATED]
+ncn-s003 is being joined to spire.
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100    87  100    54  100    33    781    477 --:--:-- --:--:-- --:--:--  1279
+Warning: Permanently added 'ncn-s003,10.252.1.4' (ECDSA) to the list of known hosts.
+[TRUNCATED]
+component name (xname) validation has been enabled.
+```
+
+### Validation
+
+To validate that xname validation is enabled, request a test JWT using the
+`heartbeat-spire-agent` command. The token shown will include an xname.
+
+```bash
+ncn# /usr/bin/heartbeat-spire-agent api fetch jwt -socketPath=/root/spire/agent.sock -audience test | head -n1
```

Comment:
```suggestion
/usr/bin/heartbeat-spire-agent api fetch jwt -socketPath=/root/spire/agent.sock -audience test | head -n1
```

we should try to not put in "ncn#" or anything prior to a command that breaks copy/paste functionality of github